Automattic/jetpack
# VideoPress: Match modal style with latest gutenberg version

Fixes #26619

#### Changes proposed in this Pull Request:

* Adds modifications made on the WordPress `Modal` component after the latest release, scoped to the VideoPress package

#### Other information:


#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Go to the VideoPress dashboard
* Click on the trash icon on a video's quick actions to open the Delete modal
* Check that there is no border on the title

Before (JN) | Before (local dev) | After
----------------|-------------------------|-------
![2022-10-06_15-19-24](https://user-images.githubusercontent.com/8486249/194389231-17074785-d368-4f62-9c14-23dcfce4b631.png) | ![2022-10-06_15-18-44](https://user-images.githubusercontent.com/8486249/194389234-b65bc1dd-e458-47a6-a5a6-dca8c7d77bca.png) | ![2022-10-06_15-20-47](https://user-images.githubusercontent.com/8486249/194389400-c504a0b9-fdf5-46fc-957f-60476cab0be2.png)
